tools/wadlto/jaxrs/src/test/java/org/apache/cxf/tools/wadlto/jaxrs/JAXRSContainerTest.java

```diff
@@ -176,7 +178,7 @@ public void testThrows() throws Exception {
         ClassCollector cc = context.get(ClassCollector.class);
         assertEquals(2, cc.getServiceClassNames().size());
 
-        final Map<String, Class<?>[]> methods = new HashMap<>();
+        final Map<String, Class<?>[]> methods = new LinkedHashMap<>();
```

Comment:
There is no dependency on ordering anywhere in the test (AFAICT) @Derrick2000 please clarify where the flakyness was observed